recipes/allure-pytest/meta.yaml

```diff
@@ -0,0 +1,121 @@
+{% set name = "allure-python" %}
+{% set version = "2.13.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/allure-framework/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9db4db2133a52e48af043fbb3bb920a77886f64a6a1c8cd56adee41828487fc6
+
+build:
+  noarch: python
+  number: 0
+
+outputs:
+  - name: allure-python-commons
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
```

Comment:
This won't work since noarch is only built on linux. In this case, you can simply rempve the windows stuff.

Reply:
ahhh, makes sense! yay less code!